### DISCUSSION TITLE
Supply Chain Greenhouse Gas Emission Factors v1.2 - 2019 US GHG data

### DISCUSSION THREAD

**Discussion by jvgregg** [USEPA/supply-chain-factors#39] [Ideas]
This new release is based on 2019 US GHG data and factors are commodity-based and given per 2021 USD. Can someone explain what does the difference in these years mean?

In addition, does these values will need to be adjusted for inflation? For instance, using the CPI of 2021 and adjusting it to CPI of 2022
e.g.: 
SEFi$2022 = SEFi$2021 * CPIi,2021/CPIi,2022

Thanks!

**Comment by WesIngwersen** [comment]
@jvgregg there has always been a difference in the year in the denominator and the GHG data in the Factors - reason being the denominator needs to match as close to possible to spend data year, which is generally from a very recent year.  These year differences and adjustments are made based on established theory and with great care and described in various places - like in the[ USEEIO v2.0](https://www.nature.com/articles/s41597-022-01293-7#Sec2) paper more recently,  The goal is to represent current conditions with the most recently available data which is different data years for different dataset.  It like for instance Google Maps - they show the whole world by stiching together data carefully from different satellite images or street images from different time periods.

We didn't use 2022 USD in the denominator because the BEA's chain-price index for gross output at the most detailed level ~400 commodities, were only available for 2021.  They have [2022 data available at a more aggregate level.](https://apps.bea.gov/iTable/?reqid=150&step=2&isuri=1&categories=gdpxind#eyJhcHBpZCI6MTUwLCJzdGVwcyI6WzEsMiwzLDNdLCJkYXRhIjpbWyJjYXRlZ29yaWVzIiwiR2RweEluZCJdLFsiVGFibGVfTGlzdCIsIjE4Il0sWyJTY2FsZSIsIjAiXSxbIkZpcnN0X1llYXIiLCIyMDIwIl0sWyJMYXN0X1llYXIiLCIyMDIyIl0sWyJSb3dzIixbIjIyUiIsIjMiLCI0IiwiNSIsIjYiLCI3IiwiOCIsIjkiLCIxMCIsIjExIiwiMTIiLCIxMyIsIjE0IiwiMTUiLCIxNiIsIjE3IiwiMTgiLCIxOSIsIjIwIiwiMjEiLCIyMiIsIjIzIiwiMjQiLCIyNSIsIjI2IiwiMjciLCIyOCIsIjI5IiwiMzAiLCIzMSIsIjMyIiwiMzMiLCIzNCIsIjM1IiwiMzYiLCIzNyIsIjM4IiwiMzkiLCI0MCIsIjQxIiwiNDIiLCI0MyIsIjQ0IiwiNDUiLCI0NiIsIjQ3IiwiNDgiLCI0OSIsIjUwIiwiNTEiLCI1MiIsIjUzIiwiNTQiLCI1NSIsIjU2IiwiNTciLCI1OCIsIjU5IiwiNjAiLCI2MSIsIjYyIiwiNjMiLCI2NCIsIjY1IiwiNjYiLCI2NyIsIjY4IiwiNjkiLCI3MCIsIjcxIiwiNzIiLCI3MyIsIjc0IiwiNzUiLCI3NiIsIjc3IiwiNzgiLCI3OSIsIjgwIiwiODEiLCI4MiIsIjgzIiwiODQiLCI4NSIsIjg2IiwiODciLCI4OCIsIjg5IiwiOTAiLCI5MSIsIjkyIiwiOTMiLCI5NCIsIjk1IiwiOTYiLCI5NyIsIjk4IiwiOTkiLCIxMDAiLCIxMDEiLCIxMDIiLCIxMDMiXV0sWyJTZXJpZXMiLCJBIl0sWyJDb2x1bW5zIixbIjMiXV1dfQ==)

That is consistent with the data source we use and I would recommend using that if you want to inflate to 2022.
